README.md

```diff
@@ -32,6 +32,7 @@ configured, take a look at the EarlGrey API and start writing your own tests.
   * [Backward Compatibility](https://github.com/google/EarlGrey/tree/master/docs/backward-compatibility.md)
   * [Install and Run](https://github.com/google/EarlGrey/tree/master/docs/install-and-run.md)
   * [API](https://github.com/google/EarlGrey/tree/master/docs/api.md)
+  * [Cheat Sheet](docs/cheatsheet/cheatsheet.png)
```

Comment:
Just a thought:
I am wondering if there is a free service that would take an HTML and output the rendered html as png which we can can be directly linked to.